tests/e2e/script_test.go

```diff
@@ -552,9 +552,13 @@ func setAnnotation(h *create.Harness, obj *unstructured.Unstructured, k, v strin
 	patch.SetGroupVersionKind(obj.GroupVersionKind())
 	patch.SetNamespace(obj.GetNamespace())
 	patch.SetName(obj.GetName())
-	annotations := map[string]string{
+	newAnnotations := map[string]string{
 		k: v,
 	}
+	annotations := patch.GetAnnotations()
```

Comment:
newAnnotations has only 1 entry, so no real point in ranging across the map.
Instead you could replace the for loop with
```golang
annotations[k] = v
```